recipes-debian/xorg-util/util-macros_1.19.2.bb

```diff
@@ -0,0 +1,28 @@
+# base recipe: meta/recipes-graphics/xorg-util/util-macros_1.19.2.bb
```

Comment:
@yuwata The file name recipes-debian/xorg-util/util-macros_1.19.2.bb should use _debian suffix doesn't it?

Reply:
Ouch! Fixed.